pages/profile-setup/components/ProfileSetupForm/ProfileSetupForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { useRouter } from "next/router";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { AxiosError } from "axios";
+import classNames from "classnames/bind";
+import { useForm } from "react-hook-form";
+
+import { postProfileSetup } from "@/apis/apis";
+import Input from "@/components/Input";
+import { PAGE_PATH } from "@/constants/pagePath";
+
+import FormSchema from "./FormSchema";
+import styles from "./ProfileSetupForm.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ProfileSetupFormProps {
+  accountId: number;
+  email: string;
+}
+
+export interface FieldValuesType {
+  nickname: string;
+  phoneNumber: string | null;
+}
+
+export default function ProfileSetupForm({ accountId, email }: ProfileSetupFormProps) {
+  const router = useRouter();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FieldValuesType>({
+    mode: "all",
+    defaultValues: { nickname: "", phoneNumber: null },
+    resolver: zodResolver(FormSchema),
+  });
+
+  const onSubmit = async (data: Omit<FieldValuesType, "accountId">) => {
+    const formData = { ...data, accountId: Number(accountId) };
+
+    try {
+      await postProfileSetup(formData);
```

Comment:
```suggestion
      const response = await postProfileSetup(formData);
      // 유저 정보 저장, 쿠키 저장 로직들....
```
p4: 여기서 `response`에 유저 정보랑 토큰 들어오는 것 같은데 유저 정보는 `zustand` 사용하니깐 전역적으로 관리하는건 어떠신가요??
그리고 받아온 토큰은 쿠키 같은걸로 저장하는 로직도 필요할 것 같아요!